eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -17,6 +17,15 @@ steps:
     inputs:
       useGlobalJson: true
       performMultiLevelLookup: true
+  - task: UseDotNet@2
+    condition: and(succeeded(), ne(variables['Agent.OS'], 'Windows_NT')) # Windows supports MultiLevelLookup and doesn't need explicit framework installation
+    displayName: 'Use .NET 7.0 SDK'
+    retryCountOnTaskFailure: 3
+    inputs:
+      # AspNetCore runtime pack can't be installed outside of SDK and we need it for integration tests
+      packageType: sdk
+      performMultiLevelLookup: true
+      version: "7.0.x"
   - task: UseDotNet@2
     condition: and(succeeded(), ne(variables['Agent.OS'], 'Windows_NT')) # Windows supports MultiLevelLookup and doesn't need explicit framework installation
     displayName: 'Use .NET 6.0 SDK'
```

Comment:
I assume you can remove 3.1 now as well based on our teams conversation.

Reply:
Not sure which conversation you're talking about, but in one thread I mentioned that track 1 still needs netcoreapp3.1.

Reply:
Ok maybe I misread that thread. If track 1 still needs this, then we need to file and issue to fix that still as .NET Core 3.1 is no longer supported.

Reply:
I opened https://github.com/Azure/azure-sdk-for-net/issues/33644 to track.